index.js

```diff
@@ -205,7 +211,11 @@ var TruffleContractSchema = {
     // update schema version
     normalized.schemaVersion = pkgVersion;
 
-    return normalized;
+    if (options.validate) {
```

Comment:
This is a nice feature. Are you using it anywhere outside of `truffle-contract-schema`? A good test for this could be one that has an invalid ABI, and therefore `normalize` would error if `validate == true` (meaning it must have called `validate()` in order to error).